pkg/systemd/index.html

```diff
@@ -1,422 +1,295 @@
-                <span id="system-info-domain" hidden></span>
-
-                <label class="control-label" for="system_information_systime_button" translate="yes">System Time</label>
-                <div role="group">
-                    <span id="systime-tooltip">
-                        <a tabindex="0" class="systime-privileged" id="system_information_systime_button"></a>
-                    </span>
-                      <a tabindex="0" hidden id="system_information_systime_ntp_status"
-                         tabindex="0" role="button" data-toggle="tooltip"
-                         data-placement="bottom" data-html="true" >
-                      </a>
-                </div>
-
-                <label class="control-label" for="shutdown-action" translate="yes">Power Options</label>
-                <div id="shutdown-group" class="btn-group">
-                    <button class="btn btn-default shutdown-privileged" id="shutdown-action" data-action="restart"
-                        data-container="body" translate="yes">Restart</button>
-                    <button data-toggle="dropdown" class="btn
-                        btn-default dropdown-toggle
-                        shutdown-privileged"
-                        aria-labelledby="system_information_power_options_label">
-                        <i class="fa fa-caret-down pf-c-context-selector__toggle-icon" aria-hidden="true"></i>
-                    </button>
-                    <ul role="menu" class="dropdown-menu">
-                        <li class="presentation">
-                            <a tabindex="0" role="menuitem" data-action="restart" translate="yes">Restart</a>
-                        </li>
-                        <li class="presentation">
-                            <a tabindex="0" role="menuitem" data-action="shutdown" translate="yes">Shut Down</a>
-                        </li>
-                    </ul>
-                </div>
-
-                <label class="control-label" for="tuned-status-button" translate="yes">Performance Profile</label>
-                <span id="system-info-performance" hidden></span>
+  <meta charset="utf-8">
+  <title>Overview</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
 
-                <label class="control-label" for="server-pmlogger-switch" translate="yes" hidden>Store Metrics</label>
-                <div id="server-pmlogger-switch" hidden>
-                </div>
+  <link rel="stylesheet" href="../base1/patternfly.css">
+  <link rel="stylesheet" href="overview.css">
 
-                <a tabindex="0" id="system-information-enable-pcp-link" hidden>
-                    <span class="pficon pficon-info"></span>
-                    <span translate>Enable stored metrics…</span>
-                </a>
+  <script type="text/javascript" src="../base1/cockpit.js"></script>
+  <script type="text/javascript" src="../base1/jquery.js"></script>
```

Comment:
The .js already `import`s jquery -- do we really need to do it again here? (I don't understand this stuff enough)

Reply:
Actually we need it - base1 directory is not resolved through webpack as a modules directory so we can't just import stuff from there.
This raises to me another question though, we we don't just import from jquery, which is a normal modules under node_modules and we build that custom jquery.js in the Makefile again :(?

Reply:
I think that's a leftover from the early package architectures where packages were not supposed to be that independent from each other. So essentially the `import` in the .js is just to satisfy eslint and such, and the "real" import happpens here? That's fine for now. Let's not get distracted by this too much for now.

In the long term, we'll remove the need for jQuery step by step anyway, and then this solves itself.